packages/dds/tree/src/tree/visitDelta.ts

```diff
@@ -184,10 +184,14 @@ function secondPass(delta: Delta.MarkList, props: PassProps): void {
             const type = mark.type;
             switch (type) {
                 case Delta.MarkType.ModifyAndDelete:
-                case Delta.MarkType.Delete:
                 case Delta.MarkType.ModifyAndMoveOut:
+                    // Handled in the first pass
```

Comment:
Consider updating/supplementing the test cases for `visitDelta` to prevent regressions here.